### ISSUE TITLE
Model instance creation with predefined data

### ISSUE THREAD

**Issue by asaf** [asaf/nodejs-model#3]
`Model.create()` should accept two params `Model.create(object, accessibility)`

This will instantiate a new model instance with predefined data which exists in the supplied object.

accessibility ensures what properties should be copied from the object, can be an array of tags or \* that matches all properties.
